cmd/devstream/install.go

```diff
@@ -18,6 +19,13 @@ var installCMD = &cobra.Command{
 
 func installCMDFunc(cmd *cobra.Command, args []string) {
 	conf := config.LoadConf(configFile)
+
+	// init before installation
```

Comment:
Init before install if the user didn't run init explicitly. Documented here: https://github.com/merico-dev/stream/issues/17.